maps/Collection.json

```diff
@@ -23,6 +23,10 @@
       "description": "Citation for the resource, including identifier, label, version, and a persistent URL to the object with SDR at the very least.",
       "type": "string"
     },
+    "currentVersion": {
+      "description": "If this is the current version (most recent version) for the resource.",
+      "type": "boolean"
```

Comment:
I'm a bit curious about the expected usage for this.  Would this be an attribute we persist?  What if someone sends `currentVersion: false` in an update?